### ISSUE TITLE
Pressing "cancel" on "move" file,  it delete the file

### ISSUE THREAD

**Issue by xion87** [pterodactyl/panel#1028]
Panel or Daemon: Both?
Version of Panel/Daemon: 0.7.3/0.5.4
Server's OS: Ubuntu 16 LTS
Your Computer's OS & Browser: Chrome/Windows 10

When i use "Move" command on file manager and press "Cancel" it delete the file instead to reject the operation, pressing "Esc" instead it "fake delete" the file until the refresh of page
![image](https://user-images.githubusercontent.com/3611852/37098432-23e714c0-221e-11e8-8779-6091bb5d2d5f.png)

**Comment by schrej** [comment]
Is this just a visual glitch or does it actually remove the file?

**Comment by xion87** [comment]
If you press cancel it delete the file (for real)
if you press esc it fake delete this file (reloading the page it appear again)

**Comment by DaneEveritt** [comment]
I am unable to reproduce this on both Chrome and Safari.

**Comment by xion87** [comment]
I tried again:
- create a file called test.yml put some text into it
- use move tool on that file, click cancel
- the file will gone, tryed also to look into sftp, is not here anymore.

**Comment by DaneEveritt** [comment]
I still cannot reproduce this.

**Comment by xion87** [comment]
btw i was using the italian translation maybe is related?

**Comment by DaneEveritt** [comment]
Theres no reason the language setting would have anything to do with that, but I went ahead and changed it, still unable to replicate this issue. In addition there is absolutely 0 logic in the move function that would delete a file, or otherwise move it to a location in which it would not show up somewhere in the file manager.

**Comment by xion87** [comment]
at least is fixed now for me for any strange reason,

**Comment by DaneEveritt** [comment]
I'm going to close this and assume it was a really strange fluke in the caching mechanics. I'm working on fixing the caching behavior on the Panel as well to avoid that ever being a potential issue down the road.

**Comment by Arsoth** [comment]
I know this is closed, but I just had the same thing occur to me while attempting... not to move a .zip file I suppose?

Panel or Daemon: Both I guess?
Version of Panel/Daemon: Panel: 0.7.6, Daemon: 0.5.5
Server's OS: Debian 9 "Stretch"
Your Computer's OS & Browser: Mac OS 10.12.6 and Firefox Nightly 61.0a1

I selected move, then attempted to cancel (honestly can't remember if it was the cancel button or the escape key, but I assume the button), and the .zip no longer was shown on the filesystem. 
Verified by checking through SSH; the file may as well never have existed.

I am unable to reproduce as of this time, but the conditions appeared to be the same as the prior bug report, so I thought I'd throw in my error as well.
